### ISSUE TITLE
Trying to send from mac

### ISSUE THREAD

**Issue by SpencerBoyd** [BTCPrivate/archived-full-node-wallet#30]
I'm getting the error "___ does not appear to be a valid BTCP address. BTCP addresses start with n1, nx, or zz!" I thought they started with b1?

**Comment by Pega88** [comment]
@SpencerBoyd please refer to #27, could it be the same? Are you confident you're running 1.0.1?

**Comment by SpencerBoyd** [comment]
@Pega88 Thanks, I missed the update to 1.0.1. Just updated and while the error changed to "... addresses start with b1, bx, or zk". The tx still wont send, even though the address does start with b1.

**Comment by Pega88** [comment]
@SpencerBoyd could you screenshot the sending screen, potentially obfuscating the last parts of the addresses & all amounts? Kinda like this:
<img width="714" alt="screen shot 2018-03-04 at 20 50 26" src="https://user-images.githubusercontent.com/1451524/36949825-c67abe36-1fed-11e8-9dc3-0e5f73133db3.png">

**Comment by SpencerBoyd** [comment]
Here you go. It says 1.0.0 at the top but when I check the about section it verifies 1.0.1. 

<img width="969" alt="screen shot 2018-03-04 at 12 06 20 pm" src="https://user-images.githubusercontent.com/7565676/36950046-5445bbf6-1fa5-11e8-81bb-1fc8cbda1677.png">

**Comment by fokep** [comment]
I just have to ask. Have you tried restarting your computer?

**Comment by SpencerBoyd** [comment]
I have, a few times just to be sure.

**Comment by fokep** [comment]
Could you also try to [clear java cache](https://www.java.com/en/download/help/mac_java_cache.xml) and verify that you have the last [version](https://www.java.com/en/download/help/version_manual.xml)?

**Comment by Pega88** [comment]
@SpencerBoyd would you be willing to share the 'to' address with me, so i can try send some myself/ detect any issue? is it with just this single address, or multiple? you can DM me in the [BTCP discord](https://discord.gg/46U2p4m) (Pega88) 
 
PS: correct, we forgot to update the UI version number with 1.0.1, still shows 1.0.0 for the 1.0.1

**Comment by SpencerBoyd** [comment]
I tried to send to a few different addresses and the issue seems to be with desktop wallet itself. I went ahead and saved the the wallet.dat, cleared java cache, deleted all BTCP files and reinstalled. I am currently downloading the blockchain again. 

I transferred out of mac electrum to windows electrum and installed fresh, just incase there was an issue with those addresses. Once the blockchain is ready I'll try again and let you all know what happens. I appreciate the support and thanks for the Discord link!
